src/sentry/buffer/redis.py

```diff
@@ -140,6 +140,11 @@ def incr(self, model, columns, filters, extra=None):
         pipe.zadd(pending_key, time(), key)
         pipe.execute()
 
+        metrics.incr('buffer.incr', skip_internal=True, tags={
```

Comment:
I'd send just:

```python
metrics.incr('buffer.incr', instance=':'.join((model.__module__, model.__name__)), skip_internal=True)
```

Then you'll get an `instance` type as `sentry.models.user:User` which is a more, full path to the model. Unless there's a reason you want them actually split into separate tags, then ignore me.

Reply:
I'm not too big on our `instance` tag. I think that having both makes it easier to read in graphs, and in the eventuality that we're only coming out of one module, separating by model will make this more digestible.